contribute/e2e_testing_environment/README.md

```diff
@@ -19,7 +19,7 @@ This framework is made up by two important components:
 
 ## The local Kubernetes cluster for testing
 
-The [`hack/setup-cluster.sh`](../../../hack/setup-cluster.sh) bash script is
+The [`hack/setup-cluster.sh`](/hack/setup-cluster.sh) bash script is
```

Comment:
The proposed change doesn't work in some rendering systems. The right change is the following:

```suggestion
The [`hack/setup-cluster.sh`](../../hack/setup-cluster.sh) bash script is
```